tables/rw-rw-g1.utb

```diff
@@ -0,0 +1,135 @@
+# liblouis: Kinyarwanda Grade 1 Braille
+#
+#-name: Kinyarwanda Grade 1 Braille
```

Comment:
The "name" tag should contain the name of the braille code in the native language.

Reply:
Removed the field for now. You can add it back if you want.